indico/modules/rb_new/controllers/backend/common.py

```diff
@@ -16,16 +16,26 @@
 
 from __future__ import unicode_literals
 
+from marshmallow import fields as mmfields
```

Comment:
i think the fields from webargs also contains Field (it basically re-exports everything from marshmallow.fields)